frontend/lib/src/components/widgets/DataFrame/ColumnMenu.test.tsx

```diff
@@ -121,4 +122,26 @@ describe("DataFrame ColumnMenu", () => {
       expect(defaultProps.onCloseMenu).toHaveBeenCalled()
     })
   })
+
+  describe("autosize functionality", () => {
```

Comment:
would it somehow be possible to test that autosizing actually works? Maybe by manually setting the width of a column to a certain value, then click on the `Autosize` button and then compare the widths? Could also be done in an e2e test